parts/k8s/openshiftmasterscript.sh

```diff
@@ -5,9 +5,11 @@
 
 SERVICE_TYPE=origin
 IMAGE_BASE=openshift/origin
+IMAGE_PREFIX=openshift/origin
 if [ -f "/etc/sysconfig/atomic-openshift-node" ]; then
 	SERVICE_TYPE=atomic-openshift
 	IMAGE_BASE=registry.reg-aws.openshift.com:443/openshift3/ose
+	IMAGE_PREFIX=registry.reg-aws.openshift.com:443/openshift3
```

Comment:
on the right lines adding IMAGE_PREFIX but I'd like to see IMAGE_BASE go away and redefine existing uses of it in terms of IMAGE_PREFIX
if it's necessary to add a separate env var to hold the 'ose' prefix, that's fine; it's the duplication of IMAGE_PREFIX and IMAGE_BASE that I don't like